Website/src/rendering/src/components/ShopCommon/QuantityInput.tsx

```diff
@@ -1,20 +1,22 @@
 import { PriceSchedule, RequiredDeep } from 'ordercloud-javascript-sdk';
-import { ChangeEvent, useState } from 'react';
+import { ChangeEvent, FocusEvent, useState } from 'react';
 
-interface ProductQuantityInputProps {
+interface QuantityInputProps {
+  controlId: string;
   priceSchedule: RequiredDeep<PriceSchedule>;
   label?: string;
   disabled?: boolean;
-  quantity: number;
+  quantity: number | string;
   onChange: (quantity: number) => void;
 }
 
-const ProductQuantityInput = ({
+const QuantityInput = ({
+  controlId,
   priceSchedule,
   disabled,
   quantity,
   onChange,
-}: ProductQuantityInputProps): JSX.Element => {
+}: QuantityInputProps): JSX.Element => {
   const [_quantity, setQuantity] = useState(quantity);
```

Comment:
What is the reason for using some variables prefixed with _ in this file? Is it to denote that they should be private or something else?

Reply:
The name `quantity` is used in the props as well, so I decided to prefix the state with _. The others are indeed unneccessary and I'll remove them.

Reply:
Sounds good, I'll let @jflheureux give the final approval. Moving the task back to Test Ready.